android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -594,4 +596,11 @@ public String getApiName()
 	{
 		return "Ti.UI.Window";
 	}
+
+	private void logDeprecatedPropertiesWarning() {
+		Log.w("Deprecated properties", "These properties are deprecated as of SDK 6.1.0: " + TiC.PROPERTY_ACTIVITY_ENTER_ANIMATION + " and " + TiC.PROPERTY_ACTIVITY_EXIT_ANIMATION);
```

Comment:
How does a log look in the compiled version? And please add a note to what to use instead.